apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/attendance/db/AttendanceRepository.java

```diff
@@ -9,7 +9,7 @@
 import edu.pw.apsienrollment.user.db.User;
 
 public interface AttendanceRepository extends JpaRepository<Attendance, Long> {
-    Optional<Attendance> findByIdAndToken(Long id, String token);
+    Optional<Attendance> findByIdAndTokenAndMeeting_Id(Long id, String token, Long meetingId);
```

Comment:
I don't like underscore in method name, but I assume Attendance has meeting_id field and Spring is not able to deduce its name without underscore?

Reply:
Attendance has the meeting and meeting has the id. The underscore is for hibernate to understand the nested property without writing extra query.

Reply:
Understood